### ISSUE TITLE
identify initial negatives in write_lab_results_to_results_project 

### ISSUE THREAD

**Issue by pbchase** [ctsit/tt_engine#11]
Expand the features of `write_lab_results_to_results_project` to identify initial negatives. 

this function should fetch its inputs from `get_lab_result_data` (name the dataframe `lab_result_data`) and results project. 

From the results project the function will read from the _UF Identifiers and Determination_ form and the _Subject Information and Results_ form. The latter form is a repeating form. It will need to read these fields:

```
record_id (aka UFID)
cleared_for_campus
instance_number
redcap_repeating_instrument_name (or some such)
acc_number (to be added)
```

Name this dataset, `lab_result_rows_in_redcap`. 

These features should already exist as a product of issue #10 

To **Identify initial negatives**, do this;

1. To **Find all initial negatives**, read `lab_result_data` and anti-join to lab_records_with_unknown_ufids by acc_number to filter out bad lab data. Then group_by ufid, filter for n=1, ungroup and filter for negatives. These are all of the initial negatives. 

2. To **identify new initial negatives**, join the above results with lab_result_rows_in_redcap by ufid=record_id. filter for cleared_for_campus == 'I'. then 

```
select(record_id = ufid) %>% 
mutate(cleared_for_campus_timestamp = today()) %>%
mutate(cleared_for_campus = 'I') %>%
```

3. Write that to the results redcap project

**Comment by pbchase** [comment]
See https://drive.google.com/open?id=1-VjOGtVozgoejUweM0awuqk3LlDHAlrS

**Comment by ChemiKyle** [comment]
I will start on this pending merge of PR #18

**Comment by pbchase** [comment]
yes, please do.  @ljwoodley or I can work on the data cleaning issues I am about to write.

**Comment by ChemiKyle** [comment]
Addressed by PR #21
